.github/workflows/generate_schema.yml

```diff
@@ -70,29 +88,41 @@ jobs:
     - name: checkout latest release
       run: |
         git checkout $SWTAG
-        
-    - name: checkout commercial plugin
+
+    - name: checkout ${{ inputs.plugin_name }} plugin
       env:
         DEPLOY_USER: ${{ secrets.DEPLOY_USER }}
         DEPLOY_TOKEN: ${{ secrets.DEPLOY_TOKEN }}
+        DIGITALSALESROOM_DEPLOY_TOKEN: ${{ secrets.DIGITALSALESROOM_DEPLOY_TOKEN }}
+        DIGITALSALESROOM_DEPLOY_USER: ${{ secrets.DIGITALSALESROOM_DEPLOY_USER }}
+        DAILY_CO_API_KEY: ${{ secrets.DAILY_CO_API_KEY }}
       run: |
-        git clone https://${DEPLOY_USER}:${DEPLOY_TOKEN}@gitlab.shopware.com/shopware/6/product/commercial.git custom/plugins/SwagCommercial
-        
+        if [ "${{ inputs.plugin_name }}" = "SwagGuidedShopping" ]; then
+          git clone --single-branch --branch 6.5-compatibility https://${DIGITALSALESROOM_DEPLOY_USER}:${DIGITALSALESROOM_DEPLOY_TOKEN}@${{ inputs.plugin_url }} custom/plugins/${{ inputs.plugin_name }}
```

Comment:
This (`--single-branch --branch 6.5-compatibility`) could be added as `input.GIT_CLONE_OPTIONS`.